### PR TITLE
Sanitize response data

### PR DESCRIPTION
**Pull request by minsulee2**
For the sake of insertion into blockchain database,
- Replace array into object
- Remove empty array and object

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: handlers/utils.js
Comment (jiyoung-an) [inline handlers/utils.js:1]: wow 👍

**Review by jiyoung-an — APPROVED** [review]
Thank you for handle this issue! 
I've learned a lot from the very clean code.

File: handlers/utils.js [hunk 1]
```diff
@@ -1,17 +1,44 @@
...
     delete data.file_ids;
+    Utils.TrimEmptyMetadataForResponseData(data);
   }
 
   static fromArrayToObjectWithTrimmingData(data) {
     data.data.forEach(v => Utils.TrimUnnecessaryDataForResponseData(v));
     data.data = { ...data.data };
   }
+
+  static fromArrayToObjectFromMessageContent(data) {
```
Comment (jiyoung-an) [inline handlers/utils.js:25]: minor) How about `fromMessageContentToObject`?
I think another good name would be fine as well. It's just that having 'from' twice in the name feels a bit awkward.

File: handlers/utils.js [hunk 2]
```diff
@@ -1,17 +1,44 @@
...
+      } else {
+        v.text.annotations = { ...v.text.annotations };
+      }
+    });
+    data.content = { ...data.content };
+    delete data.file_ids;
+    Utils.TrimEmptyMetadataForResponseData(data);
+  }
+
+  static fromArrayToObjectFromMessageList(data) {
```
Comment (jiyoung-an) [inline handlers/utils.js:38]: ditto

File: handlers/utils.js (hunk 1 above)
Comment (minsulee2) [reply handlers/utils.js]: Naming in this PR is not satisfying to me as well. Changed them by the way! Thanks :)

**Comment by minsulee2** [comment]
Thanks for the reviews! Merging now~

**Pull request merged**